### ISSUE TITLE
Release plan for v5.5 and v6

### ISSUE THREAD

**Issue by anboo44** [activej/activej#286]
I am using activeJ v5.4.3 in production enviroment. Some issues have fixed in v5.5. So, I want to know the release date of v5.5 in order to upgrade. By the way, I'm so excited when the v6 is released. 
Please give me your plan about activeJ. Thanks

**Comment by eduard-vasinskyi** [comment]
Hi, @anboo44

ActiveJ v5.5 was released today. It is already available in the [maven repository](https://mvnrepository.com/artifact/io.activej/activej/5.5). We will be updating the website shortly to include release notes for version 5.5.

We also released a beta version of ActiveJ v6.0 (that is v6.0-beta1).
Version 6.0 is ready in general, and contains great refactorings and other changes. This version is basically ready to be switched on. After testing, we will do a release within the next few weeks
